src/org/zaproxy/zap/extension/scripts/files/scripts/templates/extender/getInputsFromuser.py

```diff
@@ -0,0 +1,84 @@
+"""
+Extender scripts allow you to add completely new functionality to ZAP.
+The install function is called when the script is enabled and the uninstall function when it is disabled.
+Any functionality added in the install function should be removed in the uninstall method.
+See the other templates for examples on how to do add different functionality. 
+"""
+
+"""
+This python script will take input values from user and set them as global variables
+These variables can then be accessed by other ZAP scripts like HTTP Sender, authentication scripts, etc
+"""
+import java.awt.event;
+
+from org.parosproxy.paros.view import AbstractFrame;
+from javax.swing import JLabel;
+from javax.swing import JMenuBar;
+from javax.swing import JMenu;
+from javax.swing import JMenuItem;
+from javax.swing import JFrame;
+from javax.swing import JTextField;
+from javax.swing import JButton;
+from org.zaproxy.zap.extension.script import ScriptVars;
+
+def install(helper):
+  	print('install called'); 
+	
+	frame = JFrame("Please input OCI Values")
```

Comment:
"OCI"? Should this be changed to a generic string that will make sense to a global audience? (Or maybe I'm just missing the significance of OCI)